### DISCUSSION TITLE
Add "Ethics vote"

### DISCUSSION THREAD

**Discussion by nicebread** [RESQUE-Framework/website#100] [Ideas]
- as additional CRediT role (what was your role in preparing the application for the ethics vote?)
- and/or: Checkbox whether an ethics vote was present for this study.

**Comment by angaertner** [comment]
I like the additional checkbox! (instead of additional Credit role) – IMO it is not relevant to specify the degree (lead/support/no role) for an ethics vote, but it is interesting to know whether a vote exists or not.
